examples/panda3d-viewer-play.py

```diff
@@ -27,7 +26,7 @@
 def play_sample_trajectory():
     update_rate = 60
     cycle_time = 3
-    traj = np.repeat(talos.q0, cycle_time * update_rate, axis=1)
+    traj = np.repeat(np.matrix(talos.q0).T, cycle_time * update_rate, axis=1)
```

Comment:
I don't think this is a good idea to use matrix here, as it is deprecated.

Reply:
I agree, the code was designed using NumPy Matrix.

Do you want me to rewrite it using Array?

Reply:
I have removed `np.matrix`.